src/sqlfluff/dialects/dialect_bigquery.py

```diff
@@ -2683,6 +2684,81 @@ class ExportStatementSegment(BaseSegment):
     )
 
 
+class LoadDataStatementSegment(BaseSegment):
+    """`LOAD DATA` statement.
+
+    https://cloud.google.com/bigquery/docs/reference/standard-sql/load-statements
+    """
+
+    type = "load_data_statement"
+    match_grammar: Matchable = Sequence(
+        "LOAD",
+        "DATA",
+        OneOf("INTO", "OVERWRITE"),
+        Sequence(OneOf("TEMP", "TEMPORARY", optional=True), "TABLE", optional=True),
+        Ref("TableReferenceSegment"),
+        Sequence(
+            Bracketed(
+                Delimited(
+                    OneOf(
+                        Ref("ColumnDefinitionSegment"),
+                        Ref("TableConstraintSegment"),
+                    ),
+                    allow_trailing=True,
+                )
+            ),
+            optional=True,
+        ),
+        Sequence(
```

Comment:
*question:* This snippet is taken from `CreateTableStatementSegment`. Not sure if there is a need to consolidate it into a separate segment or if duplication is acceptable.

I would keep it as it is. Open to suggestions otherwise.